configs/sst_networking-firewall-tools.yaml

```diff
@@ -10,17 +10,17 @@ data:
   - c9s
 
   packages:
-  - arptables
-  - arptables-services
   - conntrack-tools
-  - ebtables
-  - ebtables-services
   - firewall-applet
   - firewall-config
   - firewalld
   - ipset
+  - ipset-libs
   - ipset-service
-  - iptables
+  - iptables-libs
   - iptables-nft
+  - iptables-nft-services
```

Comment:
Does this include `iptables-services` ? Or are the separate?

Reply:
They are separate, iptables-services contains iptables and ip6tables services (and is variant-agnostic).